js/Utils.js

```diff
@@ -125,8 +125,19 @@ function parseDateStringOrRestoreHeightAsInteger(value) {
         // Correct date typed in a wrong format (20201225 instead of 2020-12-25)
         var restoreHeightHyphenated = value.substring(0, 4) + "-" + value.substring(4, 6) + "-" + value.substring(6, 8);
         restoreHeight = Wizard.getApproximateBlockchainHeight(restoreHeightHyphenated, Utils.netTypeToString());
+    } else if ((device == "Ledger" || device == "Trezor") && value == "" && Utils.netTypeToString() == "Mainnet") {
+        // 1522000 = 2018-03-04 (Ledger Nano S support)
+        var ledgerMinRestoreHeight = 1522000;
+        // 1697500 = 2018-11-04 (Trezor Model T support)
+        var trezorMinRestoreHeight = 1697500;
+        console.log("Restore height undefined. Changing restore height to %1.".arg(device == "Ledger" ? ledgerMinRestoreHeight : trezorMinRestoreHeight));
+        restoreHeight = (device == "Ledger" ? ledgerMinRestoreHeight : trezorMinRestoreHeight);
```

Comment:
```suggestion
        restoreHeight = (device == "Ledger" ? ledgerMinRestoreHeight : trezorMinRestoreHeight);
        console.log("Restore height undefined. Changing restore height to %1.".arg(restoreHeight);
```

Makes the code a bit simpler.